rareapi/views/Posts.py

```diff
@@ -38,21 +40,32 @@ def create(self, request):
         #Posts default to approved when created
         post.approved = True
 
-        # Try to save the new post to the database, then
-        # serialize the post instance as JSON, and send the
-        # JSON as a response to the client request
+        #extract tag ids from request and try to convert that collection to a queryset of actual tags
+        tag_Ids = request.data["tagIds"]
```

Comment:
wow this is just so picky and dumb and idk but anyway pylint is saying this should be `tag_ids` not `tag_Ids` cause that is not "proper snake casing"